st2common/st2common/util/virtualenvs.py

```diff
@@ -166,8 +166,12 @@ def install_requirements(virtualenv_path, requirements_file_path):
     exit_code, stdout, stderr = run_command(cmd=cmd, env=env)
 
     if exit_code != 0:
-        raise Exception('Failed to install requirements from "%s": %s (stderr: %s)' %
-                        (requirements_file_path, stdout, stderr))
+        def safetystring(s):
```

Comment:
Would integration tests would probably be a bit involved, so we should probably go with some middle of the ground / compromise approach and mock some things out and use a unit test.

I can look into a test some time in the next couple of days when I get a chance :)